src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,466 @@
+module Algebra.Graph.Acyclic.AdjacencyMap (
```

Comment:
From memory, all files of `src/algebra` have a header with some documentation (used after by Haddock), maybe it's worth adding one for this file (for example: https://github.com/snowleopard/alga/blob/master/src/Algebra/Graph.hs ).

Reply:
Done!